progfun/funcional_teoria/examples.html

```diff
@@ -6,210 +6,88 @@
+        <li><a class="menu-item" href="index.html">Programación Funcional </a>
             <ul>
-				<li><a class="menu-item1" href="index.html#intro">¿Qué es?</a></li>
+                <li><a class="menu-item1" href="index.html#intro">Motivación</a></li>
                 <li><a class="menu-item1" href="index.html#history">Historia</a></li>
-                <li><a class="menu-item1" href="index.html#evolution">Evolución</a></li>
+                <li><a class="menu-item1" href="index.html#lambdacalc">C&aacute;lculo Lambda</a></li>
                 <li><a class="menu-item1" href="index.html#advantage_disadvantage">Ventajas y desventajas</a></li>
                 <li><a class="menu-item1" href="index.html#languages">Lenguajes</a></li>
             </ul>
         </li>
-        <li><a class="menu-item" href="concepts.html">Conceptos Clave</a>
-			<ul>
-				<li><a class="menu-item1" href="concepts.html#abstraccion">Abstracción</a></li>
-                <li><a class="menu-item1" href="concepts.html#modularidad">Modularidad</a></li>
-                <li><a class="menu-item1" href="concepts.html#encapsulamiento">Encapsulamiento</a></li>
-                <li><a class="menu-item1" href="concepts.html#herencia">Herencia</a></li>
-                <li><a class="menu-item1" href="concepts.html#polimorfismo">Polimorfismo</a></li>
-            </ul>
-		</li>
-		<li><a class="menu-item" href="philosophy.html">Filosofía y Principios</a>
-			<ul>
-				<li><a class="menu-item1" href="philosophy.html#filosofia">Filosofía</a></li>
-                <li><a class="menu-item1" href="philosophy.html#principios">Principios</a></li>
-                <li><a class="menu-item1" href="philosophy.html#buenaspracticas">Buenas prácticas</a></li>
-            </ul>
-		</li>
-		<li><a class="menu-item" href="examples.html" id="">Ejemplos interactivos</a>
-			<ul>
-				<li><a class="menu-item1" href="examples.html#clases">Clases</a></li>
-				<li><a class="menu-item1" href="examples.html#encapsulamiento">Encapsulamiento</a></li>
-				<li><a class="menu-item1" href="examples.html#herencia">Herencia</a></li>
-				<li><a class="menu-item1" href="examples.html#polimorfismo">Polimorfismo</a></li>
-			</ul>
-		</li>
-		
-		<li><a class="menu-item" href="applications.html">Aplicaciones</a></li>
-        <li><a class="menu-item" href="slides.html">Presentaciones y referencias</a>
-			<ul>
-				<li><a class="menu-item1" href="slides.html#slides">Presentaciones</a></li>
-				<li><a class="menu-item1" href="slides.html#referencias">Referencias y bibliografía</a></li>
-			</ul>
-		</li>
+        <li><a href="concepts.html" class="menu-item">Conceptos</a></li>
+        <li><a href="examples.html" class="menu-item">Ejemplos</a></li>
+        <li><a href="applications.html" class="menu-item">Aplicaciones</a></li>
+
+        <li><a class="menu-item" href="test.html">Test your knowledge!</a></li>
+        <li><a class="menu-item" href="/../../../index.html">Volver a Index</a></li>
```

Comment:
please change path from `/../../../index.html` to `../../index.html` right now it is broken.

Reply:
Fixed broken links.